Makefile

```diff
@@ -40,6 +40,7 @@ RESOLVE_TAGS_PACKAGE = $(REPOPATH)/$(RESOLVE_TAGS_PATH)
 RESOLVE_TAGS_KUBECTL_DIR = ~/.kube/plugins/resolve-tags
 
 LOCAL_GAC_CREDENTIALS_PATH ?= /tmp/gac.json
+LOCAL_GKE_CLUSTER_NAME ?= test-cluster-2
```

Comment:
This change seems unrelated to the pull description. Do you mind moving this into a second PR so that it can be reverted separately if needed?

Reply:
done